### DISCUSSION TITLE
Rechazo git education pack

### DISCUSSION THREAD

**Discussion by keilaayelen** [community/community#138316] [GitHub Education]
### Hi everyone,

Rechazaron mi solicitud, ya rellene el campo de facturacion como dicen en algunos foros y active mi autentificacion en 2 pasos, como hago para volver a mandar una solicitud para acceder al education pack?

**Comment by davevad93** [comment]
Hi @keilaayelen ,

By going to the [application page](https://education.github.com/discount_requests/application), you'll be able to resubmit your request to get the `Student Developer Pack`, make sure you have fixed everything necessary to avoid being rejected again.

For more info, see the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).
